docs/rfc/active/000-ml_models/README.md

```diff
@@ -0,0 +1,121 @@
+
+A machine learning model URN will look like below:
+```
+urn:li:mlModel:(<<platform>>,<<name>>,<<fabric>>)
+```
+A machine learning feature will be uniquely identified by it's name and the namespace this feature belongs to.
+A machine learning feature URN will look like below:
+```
+urn:li:mlFeature:(<<namespace>>,<<name>>)
+```
+
+### Entities
+There will be 2 top level GMA [entities](../../../what/entity.md) in the design: ML models and ML features.
+It's important to make ML features as a top level entity because ML features could be shared between different ML models.
+
+### ML Model metadata
+- Model properties: Basic information about the ML model
+  - Model date
+  - Model desription
+  - Model version
+  - Model type: Basic model architecture details e.g. if it is Naive Bayes classifier, Convolutional Neural Network, etc
+  - ML features used for training
+  - Hyperparameters of the model, used to control the learning process
+  - Tags: Primarily to enhance search and discovery of ML models
+- Ownership: Users who own the ML model, to help with directing questions or comments about the model.
+- Intended Use
+  - Primary intended use cases
+  - Primary intended user types
+  - Out-of-scope use cases
+- Model Factors: Factors affecting model performance including groups, instrumentation and environments
+  - Relevant Factors: Foreseeable factors for which model performance may vary
+  - Evaluation Factors: Factors that are being reported
+- Metrics: Measures of model performance being reported, as well as decision thresholds (if any) used.
+- Training Data: Details on datasets used for training ML Models
+  - Datasets used to train the ML model
+  - Motivation behind choosing these datasets
+  - Preprocessing steps involved: crucial for reproducibility
+  - Link to the process/job that captures training execution
+- Evaluation Data: Mirrors Training Data.
+- Quantitative Analyses: Provides the results of evaluating the model according to the chosen metrics by linking to relevant dashboard.
+- Ethical Considerations: Demonstrate the ethical considerations that went into model development, surfacing ethical challenges and solutions to stakeholders.
+- Caveats and Recommendations: Captures additional concerns regarding the model
+  - Did the results suggest any further testing?
+  - Relevant groups that were not represented in the evaluation dataset
+  - Recommendations for model use
+  - Ideal characteristics of an evaluation dataset
+- Institutional Memory: Institutional knowledge for easy search and discovery.
+- Status: Captures if the model has been soft deleted or not.
+- Cost: Cost associated with the model based on the project/component this model belongs to.
+- Deprecation: Captures if the model has been deprecated or not.
```

Comment:
Does it makes sense to have a visual representation of the High level MLModel schema like this?
![MLModelSchema](https://user-images.githubusercontent.com/12974850/90677106-85d02e80-e222-11ea-8952-216ecb261fca.png)

Reply:
I love the idea, let me add it.

Reply:
Sorry, I might have to drop this suggestion. Tried creating one but the list of aspects and associated fields is huge and the diagram looks cluttered.